.directory

```diff
@@ -0,0 +1,4 @@
+[Dolphin]
```

Comment:
Please remove this file from the commit!

Reply:
My kde installation must have sucked me up.